src/Common/EncodingDataBase.ts

```diff
@@ -21,6 +21,10 @@ abstract class EncodingDataBase<T extends number> {
     return result;
   }
 
+  public get entries() {
```

Comment:
Not really a problem, but also not necessary, a call to `lookup(SemaphoreEncoding.None)` should yield all the entries.